docs/rule/require-lang-attribute.md

```diff
@@ -54,6 +54,15 @@ This rule **allows** the following:
 
 Add the `lang` attribute to the `app/index.html` file in your Ember app. If you use an internationalization addon like `ember-intl`, note that this will not conflict with that addon.
 
+## Configuration
+
+- boolean -- if `true`, default configuration is applied
+  (`validateValues: false`), see below for details
+
+- object -- containing the following property:
+  - boolean -- `validateValues` -- if `true`, the rule checks whether the value in the `lang` attribute is a known IETF's BCP 47 language tag
+    (default: `false`)
```

Comment:
We could indicate here that we should enable this by default in the next major release as a reminder:

```suggestion
    (default: `false`) (TODO: enable by default in next major release)
```